server/pkg/asset/upload.go

```diff
@@ -0,0 +1,32 @@
+package asset
+
+import (
+	"time"
+)
+
+type Upload struct {
+	uuid      string
+	project   ProjectID
```

Comment:
```suggestion
	project   ProjectID
	user      UserID
```

It is better to add userID to Upload and use it on MongoDB query to filter uploads by userID